integration/utils.mjs

```diff
@@ -0,0 +1,13 @@
+export const waitForAndClickElement = async (page, query) => {
+  await waitForElement(page, query);
+  const element = await page.$(query);
+  await element.click();
+};
+
+export const waitForElement = async (page, query) => {
+  await page.waitForSelector(query, {
+    visible: true,
+  });
+};
+
+export const sleep = (ms) => new Promise((r) => setTimeout(r, ms));
```

Comment:
Not sure if it's good to define our own sleep function using `setTimeout()`.

Was there any issue with using the built-in `await page.waitFor(x)`?

Reply:
Oh because `Page.waitFor()` no longer exists on the Puppeteer documentation. 

Next closest thing to "wait for x milliseconds" is `Page.waitForTimeout()`, but its obsolete. So I followed their note on https://pptr.dev/api/puppeteer.page.waitfortimeout to use `new Promise(r => setTimeout(r, milliseconds));`